lib/capistrano/tasks/nginx.rake

```diff
@@ -38,18 +43,18 @@ namespace :nginx do
   desc 'Enable nginx site'
   task :enable_site do
     on roles fetch(:nginx_roles) do
-      execute :ln, "-sf",
-        "#{fetch(:nginx_path)}/sites-available/#{fetch(:application)}.conf",
-        "#{fetch(:nginx_path)}/sites-enabled/#{fetch(:application)}.conf"
+      sudo :ln, "-sf",
+        "#{fetch(:nginx_path)}/#{fetch(:nginx_sites_available_subfolder)}/#{fetch(:application)}.conf",
+        "#{fetch(:nginx_path)}/#{fetch(:nginx_sites_enabled_subfolder)}/#{fetch(:application)}.conf"
     end
     invoke :'nginx:reload'
   end
 
   desc 'Disable nginx site'
   task :disable_site do
     on roles fetch(:nginx_roles) do
-      config_link = "#{fetch(:nginx_path)}/sites-enabled/#{fetch(:application)}.conf"
-      execute :unlink, config_link if test "[ -f #{config_link} ]"
+      config_link = "#{fetch(:nginx_path)}/#{fetch(:nginx_sites_enabled_subfolder)}/#{fetch(:application)}.conf"
+      sudo :unlink, config_link if test "[ -f #{config_link} ]"
```

Comment:
You shouldn't use sudo. Instead you should give the user that commands run as permission to access and modify the necessary files and directories.

Reply:
> Are there default installations of nginx where the folders are suffixed with .d?

What do you mean?